### PR TITLE
GEOD-514 - Convert to using Page Objects in our e2e tests

### PR DESCRIPTION
**Pull request by oehm-smith**
* SelectSite for this one

### CODE REVIEW

**Comment by oehm-smith** [comment]
Hold off from reviewing.

File: src/e2e/specs/select-site.pageobject.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { element, by, ElementHelper, ElementFinder } from 'protractor';
+
+export class SelectSitePage {
+    readonly url:string = '/';
+    readonly searchBox: ElementFinder = element(by.css('sd-select-site input[name="searchText"]'));
```
Comment (lbodor) [inline src/e2e/specs/select-site.pageobject.ts]: Is `by.binding` an option here? Some guidelines say we should prefer protractor locators. See: https://github.com/CarmenPopoviciu/protractor-styleguide#rule-07-prefer-protractor-locators-when-possible.

Comment (oehm-smith) [inline src/e2e/specs/select-site.pageobject.ts]: Nope no bindings on this page.

**Comment by lbodor** [comment]
:+1:

**Pull request merged**